### DISCUSSION TITLE
Great card - love it!

### DISCUSSION THREAD

**Discussion by BorisPocatko** [dylandoamaral/uptime-card#75] [Ideas]
It's a great and highly customizable card, kudos!
Will be maybe trying to add time stamps if the average is above a threshold on the bottom footer (around line 670 in uptime-card.ts) once I get some spare time. 

Thanks again for a great addon!

**Comment by dylandoamaral** [comment]
Hey I think you already can do that using javascript template (since v0.5.0), otherwise I will be more than happy to look your PR :D.

Thank you very much !
